apps/drupal/webpack.drupal.dev.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Drupal-specific webpack config.
+ * This is merged over top of webpack.particle.dev.js
+ */
+
+// Library Imports
+const merge = require('webpack-merge');
+const WebpackShellPlugin = require('webpack-shell-plugin');
+
+// Custom Imports
+// Import shared PROD settings because webpack.particle.prod extracts styles to its own standalone
+// CSS files. So drupal dev uses shared prod settings, in dev mode to rapidly build all assets.
+const particle = require('../../webpack.particle.prod.js');
```

Comment:
some of our imported configs have .js on the end and some don't, we should have them all consistent